release-please-config.json

```diff
@@ -11,7 +11,8 @@
       "extra-files": [
         "README.md",
         "config/manager/kustomization.yaml",
-        "website/src/partials/_install.mdx"
+        "website/src/partials/_install.mdx",
+        "website/src/static/release.txt"
```

Comment:
I think this will only work if the file has a release please version annotation, see https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files